### ISSUE TITLE
About the .train() and .eval() mode.

### ISSUE THREAD

**Issue by YongjianDeng** [backseason/PoolNet#30]
Hi,

I notice that whatever train or test your model, you chose net.eval() mode to run the whole procedure. Does that mean you use the Batchnorm layer only with its init running mean and var? Does this operation make your model have better performance through your experiments?

Thanks for your reading and I look forward to hearing from you.

**Comment by backseason** [comment]
Hi,
As the `.train()` or `.eval()` mode only affect the BN layers in our network, we freeze these layers during both training and testing.
All the parameters of BN layers are initialized with the imagenet pre-trained model.
For now our model can only run with `batch_size=1`, there is no reason to learn the params of these layers with so small a batch size.

**Comment by YongjianDeng** [comment]
> Hi,
> As the `.train()` or `.eval()` mode only affect the BN layers in our network, we freeze these layers during both training and testing.
> All the parameters of BN layers are initialized with the imagenet pre-trained model.
> For now our model can only run with `batch_size=1`, there is no reason to learn the params of these layers with so small a batch size.

Hi,

Thanks for your timely reply, I get the point now
